<launch>
	<arg name="host" default="localhost" />

	<rosparam param="/mbot_sim_node/Surrogate/front/Scene/disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
  </rosparam>

  <rosparam param="/mbot_sim_node/Surrogate/right/Scene/disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
  </rosparam>

  <rosparam param="/mbot_sim_node/Surrogate/rear/Scene/disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
  </rosparam>

  <rosparam param="/mbot_sim_node/Surrogate/left/Scene/disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
  </rosparam>

  <rosparam param="/mbot_sim_node/Surrogate/rs_front/DepthPerspective/disable_pub_plugins">
    - 'image_transport/compressedDepth'
    - 'image_transport/compressed'
    - 'image_transport/theora'
  </rosparam>

	<node name="mbot_sim_node" pkg="mbot_ros_pkgs" type="mbot_sim_node" output="screen">
		<param name="update_img_response_every_n_sec" type="double" value="0.05" />
		<param name="update_airsim_control_every_n_sec" type="double" value="0.01" />
		<param name="update_airsim_ground_truth_every_n_sec" type="double" value="1" />
		<param name="update_lidar_every_n_sec" type="double" value="0.1" />
		<param name="update_gps_every_n_sec" type="double" value="1.0" />
		<param name="host_ip" type="string" value="$(arg host)" />
	</node>

	<!-- Static transforms -->
	<include file="$(find mbot_ros_pkgs)/launch/static_transforms.launch"/>
</launch>
